docs/conversion/sakai_12_oracle_conversion.sql

```diff
@@ -833,3 +833,8 @@ ALTER TABLE SAKAI_USER MODIFY MODIFIEDON TIMESTAMP NOT NULL;
 ALTER TABLE SAKAI_USER MODIFY CREATEDON TIMESTAMP NOT NULL;
 
 -- END KNL-1566
+
+-- SAK-40528 Add index on ASN_ASSIGNMENT.CONTEXT
+ALTER TABLE ASN_ASSIGNMENT ADD INDEX `IDX_ASN_ASSIGNMENT_CONTEXT` (`CONTEXT`);
```

Comment:
seems like format is CREATE INDEX SAM_ITEMTAG_ITEMID_I ON SAM_ITEMTAG_T(ITEMID);

Reply:
@bjones86 can western help with oracle syntax here?

Reply:
No worries i updated the oracle syntax to match existing!